include/git2/remote.h

```diff
@@ -474,6 +474,24 @@ typedef int GIT_CALLBACK(git_push_negotiation)(const git_push_update **updates,
  */
 typedef int GIT_CALLBACK(git_push_update_reference_cb)(const char *refname, const char *status, void *data);
 
+/**
+ * Callback to resolve URLs before connecting to remote
+ *
+ * You need to be careful not to overflow url_resolved. If it's too small,
+ * returning a positive value equal to the size required will cause the
+ * resolution to be tried again with a buffer of the requested size.
```

Comment:
Is this comment accurate?  I don't see this behavior.

Reply:
Oops, a leftover from the old way, before I re-read buffer.h. Please disregard.